driver-core/src/main/java/com/datastax/driver/core/exceptions/ReadTimeoutException.java

```diff
@@ -42,8 +42,11 @@ public ReadTimeoutException(
     super(
         endPoint,
         String.format(
+            "Cassandra timeout during read query at consistency %s (%s). "
+                + "In case this was generated during read repair, the consistency level is not representative of the actual consistency.",
             "Cassandra timeout during read query at consistency %s (%s)",
-            consistency, formatDetails(received, required, dataPresent)),
+            consistency,
```

Comment:
another comment is that this is an unnecessary formatting change not related to the problem we want to fix. please remove.

Reply:
Thanks. Taken care of now.